node-notes-cli/app.js

```diff
@@ -0,0 +1,31 @@
+const dataObject = require('./data.json')
+const fs = require('fs');
+
+
+fs.readFile('data.json', 'utf8', (err, data) => {
+  data = JSON.parse(data);
+  if(process.argv[2] === 'read') {
+    for (var i = 1; data.nextId > i; i++) {
+      console.log(`${i}: ${data.notes[i.toString()]}`);
+    }
+  } else if(process.argv[2] === 'create') {
+    let nextId = data.nextId;
+    data.notes[nextId] = process.argv[3];
+    data.nextId++;
```

Comment:
I'd suggest making sure there _is_ a note (`process.argv[3]`) first. If the user forgot to provide one, let them know instead of creating one.